common/Vulkan/Context.cpp

```diff
@@ -676,7 +676,6 @@ namespace Vulkan
 		{
 			m_device_driver_properties.sType = VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_DRIVER_PROPERTIES;
 			*pNext = &m_device_driver_properties;
-			pNext = &m_device_driver_properties.pNext;
```

Comment:
This is here so you can chain these
It may be the end of the chain now, but may not be in the future